pkg/log/log_messages.go

```diff
@@ -70,3 +70,5 @@ const CAKC054 string = "CAKC054 Failed to delete file %s"
 const CAKC055 string = "CAKC055 Cert placement failed with the following error:\n%s"
 const CAKC056 string = "CAKC056 File %s does not exist"
 const CAKC057 string = "CAKC057 Removing file %s"
+const CAKC058 string = "CAKC058 Permissions error when checking if file exists: %s"
+const CAKC059 string = "CAKC059 Path exists but does not contain regular file: %s"
```

Comment:
@shulifink can you review these messages please?

Reply:
@shulifink , any chance you can review these log messages? Thanks!